test/functional/bundleadd/add-existing/test.bats

```diff
@@ -15,7 +15,8 @@ teardown() {
 
 @test "bundle-add an already existing bundle" {
   run sudo sh -c "$SWUPD bundle-add $SWUPD_OPTS test-bundle"
-  [ "$status" -eq 0 ]
+  echo "$status"
```

Comment:
Probably a leftover

Reply:
Not a leftover, I added it on purpose, when that echo is not there, and the test fails, you can only see in the test log that the test failed because it got an incorrect exit status, but you cannot see what the actual exit status is. So I think is better to have that value there all the time.

Reply:
oh, ok then